scss/styles/course.scss

```diff
@@ -8,6 +8,38 @@
     }
 }
 
+// Full width header layout on course pages.
+.pagelayout-course,
+.pagelayout-incourse {
+    #page.drawers #topofscroll,
+    #page-header {
+        padding-left: 0;
+        padding-right: 0;
+    }
+
+    #page {
+        overflow-x: hidden;
```

Comment:
This has got my spider sense twitching. Is there a chance we will obscure content that needs to be seen?